### PR TITLE
Add VEBA OVA deployment scripts for Knative

### PR DESCRIPTION
**Pull request by lamw**
Closes: #714
Signed-off-by: William Lam <wlam@vmware.com>

## Summary

This change simply adds a few examples for deploying VEBA OVA using Knative processor. It includes standard Knative deployment, Knative + VEBA UI and Knative + VEBA UI + Webhook example

## Pull Request Checklist
🚨 Please review the [guidelines for contributing](https://vmweventbroker.io/community) to this repository.

- [ ] Please ensure that you are making a pull request against the **Development** branch
- [ ] Please use the `WIP` keyword in the title of your PR if you are not ready for review
- [ ] Please ensure that you have opened a Github Issue if you are resolving/fixing a problem
- [ ] Please ensure that you have [signed](https://help.github.com/en/github/authenticating-to-github/signing-commits) all commits and that you have [squashed](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec) all relevant commits related to your change
- [ ] Please make sure that you have tested your change locally by successfully [building and deploying the VMware Event Broker Appliance](https://vmweventbroker.io/kb/contribute-appliance) and/or [building and deploying VMware Event Router](https://vmweventbroker.io/kb/contribute-eventrouter)
- [ ] Please include any relevant screenshots and/or output as part of your testing
- [ ] Please include any documentation updates that is applicable for your changes

## Change Type

What types of changes does your code introduce to the VMware Event Broker Appliance?

_Put an `x` in all boxes that apply_

Please check the type of change your PR introduces:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation changes
- [ ] Other (please describe):

### CODE REVIEW

File: test/deploy_veba_knative_processor.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+# Copyright 2020 VMware, Inc. All rights reserved.
```
Comment (embano1) [inline test/deploy_veba_knative_processor.sh:2]: no `pipefail` etc?

File: test/deploy_veba_knative_processor.sh [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+# SPDX-License-Identifier: BSD-2
+
+# Sample Shell Script to test deployment of VEBA w/Knative Processor
+
+OVFTOOL_BIN_PATH="/Applications/VMware OVF Tool/ovftool"
```
Comment (embano1) [inline test/deploy_veba_knative_processor.sh:7]: assumes OSX?

Comment (lamw) [reply test/deploy_veba_knative_processor.sh:9]: User would need to specify the path to their `ovftool` installation, this is merely an example. It also matches whats already there :)

**Review by embano1 — APPROVED** [review]
LGTM

**Pull request merged**